src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -41,7 +41,7 @@ import { type OnAnnotationEdit } from '../comment/types';
 
 import './ActivityFeed.scss';
 
-type Props = {
+export type ActivityFeedProps = {
```

Comment:
nit: if the reason for changing the name here is because its being exported I would probably change the name at the point of export in the index folder instead

Reply:
I don't quite see the advantage, and I think there is a disadvantage to renaming exports in terms of readability. I have a strong preference for providing a good name where I variable is defined.